app/views/spree/admin/transaction_sync_batches/index.html.erb

```diff
@@ -4,8 +4,12 @@
   <%= "Transaction Sync Batches" %>
 <% end %>
 
-<%= form_with url: admin_transaction_sync_batches_path, method: :post, local: true do %>
-  <%= submit_tag "Backfill Transactions", class: "btn btn-primary" %>
+<%= form_with url: admin_transaction_sync_batches_path, method: :post, local: true do |f| %>
+  <%= f.label :start_date %>
+  <%= f.date_field :start_date %>
```

Comment:
What do you think about providing a data/time input selector for extra granularity and storing this as a `DateTime`. I believe Solidus [has some partials](https://github.com/solidusio/solidus/blob/e878076f2ed670d07654ab6293a16588743f2fa6/backend/app/views/spree/admin/promotions/_form.html.erb#L47) for rendering date + time fields. You can see an example from promotions admin here
<img width="616" alt="Screen Shot 2022-09-29 at 5 14 20 PM" src="https://user-images.githubusercontent.com/3153035/193163630-e4af8247-17e9-4c95-8fb7-7c4f6c9d9c35.png">

Reply:
I'm not opposed to adding this feature in a new PR but I think using browser supplied date inputs is sufficient for now.